Pod/Classes/PINRemoteImageManager.m

```diff
@@ -192,11 +194,14 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
 
 - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)configuration alternativeRepresentationProvider:(id <PINRemoteImageManagerAlternateRepresentationProvider>)alternateRepProvider
 {
-    return [self initWithSessionConfiguration:configuration alternativeRepresentationProvider:nil imageCache:nil];
+    return [self initWithSessionConfiguration:configuration alternativeRepresentationProvider:nil imageCache:nil retryStrategyCreationBlock:nil];
```

Comment:
I also felt like that at first, then decided be more precise. I think it's a good idea to have in init, but it's not principal, just let me know keep it or revert back.